specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/diskRPCommon.json

```diff
@@ -360,6 +360,10 @@
         "securityDataUri": {
           "type": "string",
           "description": "If createOption is ImportSecure, this is the URI of a blob to be imported into VM guest state."
+        },
+        "performancePlus": {
+          "type": "boolean",
+          "description": "Set this flag to true to get a boost on the performance target of the disk deployed, see here on the respective performance target. This flag can only be set on disk creation time and cannot be disabled after enabled."
```

Comment:
suggest getting an aka ms link to refer here for feature details

Reply:
Talked to PM and she said we don't have one yet.